specification/ai/ModelClient/tspconfig.yaml

```diff
@@ -23,3 +23,10 @@ options:
     package-dir: "azure-ai-inference"
     package-name: "{package-dir}"
     flavor: azure
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.AI.Inference"
+    namespace: "Azure.AI.Inference"
+    clear-output-folder: true
+    model-namespace: false
+    generate-protocol-methods: false
```

Comment:
I see "Krzysztof Cwalina" <kcwalina@microsoft.com> is OOF until 7/22... he was asking why we are not generating the protocol methods. I asked if they are needed by all SDKs, and why. I don't think he answered that question (perhaps he left for vacation before he could answer). But anyway, this is where we would enable them. Okay to leave it "false" for now.